.github/workflows/ci.yml

```diff
@@ -56,21 +56,21 @@ jobs:
           poetry config http-basic.epc-power aws $CODEARTIFACT_AUTH_TOKEN
           poetry config repositories.third-party https://epc-power-us-${{ secrets.AWS_ACCOUNT_ID }}.d.codeartifact.us-east-2.amazonaws.com/pypi/third-party/
           poetry config http-basic.third-party aws $CODEARTIFACT_AUTH_TOKEN
+          poetry config installer.max-workers 1    # https://github.com/python-poetry/poetry/issues/7611
 
           poetry install
-
           poetry --version
       - name: python commands
         run: |
           set -vx
           poetry run pip --version
+          poetry run poetry-dynamic-versioning
```

Comment:
This is the old way that we are trying to remove.
```suggestion
```

Reply:
Resolved.